db/seeds/scope_limitations.rb

```diff
@@ -0,0 +1,18 @@
+require 'csv'
+
+file_path = Rails.root.join('db', 'seeds', 'scope_limitations.csv')
+
+data = CSV.read(file_path)
+data.shift
+
+data.each do |row|
+  code, meaning, description, substantive, delegated_functions = row
+  scope_limitation = ScopeLimitation.where(code: code).first_or_initialize
+  attrs = {
+    meaning: meaning,
+    description: description,
+    substantive: substantive,
+    delegated_functions: delegated_functions
+  }
+  scope_limitation.update_attributes(attrs)
```

Comment:
Would `.update` work here? `.update_attributes` seems to be an alias of  `.update` (https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attributes)

Also, I recommend using the version with the exclamation mark (`.update!`). Otherwise, if it fails, it won't raise any error.